pyclibrary/c_parser.py

```diff
@@ -1255,40 +1467,22 @@ def is_fund_type(self, typ):
         """Return True if this type is a fundamental C type, struct, or
         union.
 
-        """
-        if (typ[0][:7] == 'struct ' or typ[0][:6] == 'union ' or
-                typ[0][:5] == 'enum '):
-            return True
+        **ATTENTION: This function is legacy and should be replaced by
+        Type.is_fund_type()**
 
-        names = base_types + size_modifiers + sign_modifiers
-        for w in typ[0].split():
-            if w not in names:
-                return False
-        return True
+        """
+        return Type(typ).is_fund_type()
 
     def eval_type(self, typ):
```

Comment:
That's what I add in mind but it seems those are not loud anymore so people would miss them nonetheless. Don't bother.